packages/aws-rfdk/lib/lambdas/nodejs/lib/x509-certs/test/certificate.test.ts

```diff
@@ -273,5 +273,7 @@ test('decrypt private key', async () => {
   // THEN
   expect(decryptedKey).toEqual(expectedDecryptedKey);
   // Must have the decrypted private key
-  expect(decryptedKey).toContain('-----BEGIN RSA PRIVATE KEY-----');
+  // OpenSSL 1.0.x: -----BEGIN RSA PRIVATE KEY-----
+  // OpenSSL 1.1.x: -----BEGIN PRIVATE KEY-----
+  expect(decryptedKey).toMatch(/-----BEGIN (?:RSA )?PRIVATE KEY-----/);
```

Comment:
I was developing this on an Ubuntu 22.04 machine that uses OpenSSL 1.1.x, so I fixed this up while I was at it.